Main.java

```diff
@@ -3,74 +3,145 @@
                 }
-                if (wordCount == 0) {
-                    System.out.println("Word does not exist");
-                } else {
-                    System.out.println("Word found");
-                    System.out.println(searchKeyword + " occurs " + wordCount + " times in the file");
-                }
+            }
+            if (count == 0) {
+                ERROR_MESSAGE = "Word does not exist";
+                System.out.println(ERROR_MESSAGE);
+                Database.wordSearchFailure(textFilePath, searchKeyword, ERROR_MESSAGE);
+            } else {
+                System.out.println("Word found");
+                System.out.println(searchKeyword + " occurs " + count + " times in the file");
+                Database.wordSearchSuccess(textFilePath, searchKeyword, count);
             }
         } catch (Exception e) {
             e.printStackTrace();
         }
     }
 }
 
+class Database {
+
+    private static String CONNECTION_URL = "jdbc:mysql://localhost:3306/elixr_labs_internship";
+    private static String USER = "root";
+    private static String PASSWORD = "abhijith.sajimon@29";
+    private static String SEARCH_RESULT_SUCCESS = "Success";
+    private static String SEARCH_RESULT_FAILURE = "Failure";
+
+    public static void wordSearchFailure(String textFilePath, String searchKeyword, String errorMessage) {
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
+        } catch (ClassNotFoundException e) {
+            e.printStackTrace();
+        }
+        String insertSql = "INSERT INTO audit VALUES(?,?,now(),?,?,?)";
+        try {
+            Connection connect = DriverManager.getConnection(CONNECTION_URL, USER, PASSWORD);
+            statement = connect.prepareStatement(insertSql);
+            statement.setString(1, textFilePath);
+            statement.setString(2, searchKeyword);
+            statement.setString(3, SEARCH_RESULT_FAILURE);
+            statement.setInt(4, 0);
+            statement.setString(5, ERROR_MESSAGE);
+            statement.executeUpdate();
+            connect.close();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static void wordSearchSuccess(String textFilePath, String searchKeyword, int count) {
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
+        } catch (ClassNotFoundException e) {
+            e.printStackTrace();
+        }
+        String insertSql = "INSERT INTO audit VALUES(?,?,now(),?,?,?)";
+        try {
+            Connection connect = DriverManager.getConnection(CONNECTION_URL, USER, PASSWORD);
+            statement = connect.prepareStatement(insertSql);
+            statement.setString(1, textFilePath);
+            statement.setString(2, searchKeyword);
+            statement.setString(3, SEARCH_RESULT_SUCCESS);
+            statement.setInt(4, count);
+            statement.setString(5, null);
+            statement.executeUpdate();

```

Comment:
While the idea to keep the wordSearchSuccess and wordSearchFailure as separate methods is good, you currently have a lot of duplicated code in both. You should try to move the duplicated code into a private method that wordSearchSuccess() and wordSearchFailure() can call internally